_posts/2024-01-17-devops-and-flutterflow.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: postv2
+font: serif
+title: "Role of FlutterFlow in Modern App Development: Insights for DevOps Teams"
+description: "FlutterFlow is a visual editor for building Flutter apps. It is a good starting point to move github and use DevOps practices to build out a full app"
+date: 2023-10-17
+highlight: monokai
+image: img/devopsflutterflow/cover.png
+author: Matthew Carter
+tags: [Trust policy, GitHub, GitHub Environments, Security, OIDC]
+---
+
+# Role of FlutterFlow in Modern App Development: Insights for DevOps Teams
+
+## Introduction
+
+- A click and drag interface for building basic Flutter apps quickly
+- Good base for building out a POC or MVP
+- Can be used to build out a full app
+
+## What is FlutterFlow?
+
+- A visual editor for building Flutter apps
+- Extensive templates and pre-built components
+- Database integrations
+- Authentication
+- Export to code
```

Comment:
what can you code in? Is it easily extensible?

Reply:
exports to flutter code, can't see any alternatives to that